### ISSUE TITLE
Use readable color combinations in CLI and GUI tools

### ISSUE THREAD

**Issue by emanruse** [QubesOS/qubes-issues#9417]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### The problem you're addressing (if any)

Various Qubes tools use color combinations with poor readability.

Examples: Red text on black background in `qubes-dom-update` and `qvm-run --pass-io`, also #8949, etc.

Although the user could change the background for console, that is still not good enough. It rather confirms the problem than resolving it.

### The solution you'd like

Use readable color combinations.

Some useful tools in links here:
https://redlib.northboot.xyz/r/accessibility/comments/154wid3/red_text_on_dark_background/

### The value to a user, and who that user might be

Good readability.

### Completion criteria checklist

(This section is for developer use only. Please do not modify it.)
